account_invoice_supplier_ref_unique/tests/test_account_invoice_supplier_ref_unique.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Akretion
+# Copyright 2010 - 2014 Savoir-Faire-Linux
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.addons.account.tests.account_test_classes\
+    import AccountingTestCase
+from openerp.exceptions import ValidationError
+
+
+class TestAccountInvoiceSupplierRefUnique(AccountingTestCase):
```

Comment:
I don't think it's necessary to inherit from AccountingTestCase, you don't anything from that, and it makes the test more fragile.